src/postprocessing.py

```diff
@@ -200,4 +258,123 @@ def crop_image_center_per_class(image, h_crop, w_crop):
         cropped_prediction = class_prediction[h_start:-h_start, w_start:-w_start]
         cropped_per_class_prediction.append(cropped_prediction)
     cropped_per_class_prediction = np.stack(cropped_per_class_prediction)
-    return cropped_per_class_prediction
\ No newline at end of file
+    return cropped_per_class_prediction
+
+
+def join_score_image(image, score):
+    return (image, score)
+
+
+def get_features_for_image(image, probabilities, annotations):
```

Comment:
@apyskir I think I'd rather have this as a transformer where in transform we just execute private prepare_data and then we run _transform where a list of those feature_extractions is executed. That would make it easy to read I think. If you want to have it as a function(which is ok of course) extract chunks responsible for getting to the one object mask point and feature extractions for that object mask.

Reply:
@jakubczakon done